static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -397,36 +397,12 @@ define([
     }
 
     function adblockInUse() {
-        if (!detect.cachedAdblockInUse) {
-            var sacrificialAd = createSacrificialAd(),
-                contentBlocked = isHidden(sacrificialAd);
-            sacrificialAd.remove();
-            detect.cachedAdblockInUse = contentBlocked;
-            return contentBlocked;
-        }
-
-        return detect.cachedAdblockInUse;
-
-        function isHidden(bonzoElement) {
-            return bonzoElement.css('display') === 'none';
-        }
+        return !!window.guardian.adBlockers && window.guardian.adBlockers.generic;
```

Comment:
it should be yeah – this is just a massive guard against this code running before its gets set. it shouldn't ever, because `detect` (currently) does not run till domready

it's really a just-in-case